src/main/java/com/nti/socialmediaappcore/payload/response/JwtResponse.java

```diff
@@ -0,0 +1,65 @@
+package com.nti.socialmediaappcore.payload.response;
+
+
+import java.util.List;
+
+public class JwtResponse {
```

Comment:
Rename to e.g. UserDetailsDTO and move to dto directory (package)
Use Lombok annotation and remove getters/setters

```suggestion
@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDetailsDTO {
```